tests/test_func_call_1.dcf

```diff
@@ -0,0 +1,12 @@
+foo(int i) -> int {
+	int x = i + 1;
+	return x;
+}
+
+main() -> int {
+	int y = foo(1);
+	if (y === 2) {
+		print("I can do math");
+	}
+	return y;
```

Comment:
ok thx. (Where can I see that ./testall.sh expects me to return 0 out of curiosity?)